src/house.cpp

```diff
@@ -190,37 +190,37 @@ bool House::transferToDepot() const
 		return false;
 	}
 
-	Player* player = g_game.getPlayerByGUID(owner);
+	std::shared_ptr<Player> player = g_game.getPlayerByGUID(owner);
```

Comment:
Could this be `const auto`
```suggestion
	cont auto player = g_game.getPlayerByGUID(owner);
```